app/src/main/res/xml/settings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <PreferenceCategory
+        android:title="@string/app_name">
+
+        <SwitchPreference
+            android:key="bedtime"
+            android:title="@string/bedtime" />
+        <Preference
+            android:key="bedtime_time"
+            android:title="@string/time_to_sleep" />
+
+        <SwitchPreference
+            android:title="@string/notification_GTFO"
+            android:key="GTFO"/>
+        <ListPreference
+            android:title="@string/snooze_time"/>
```

Comment:
https://developer.android.com/reference/android/preference/ListPreference.html

chcelo by to tam asi pridat android:entries na ten array 1..15 co si vytvoril a asi nieco v kode..potom to mozno ani padat nebude... :D

Reply:
hej hej viem :D len uz sa mi to nechcelo riesiet tak som to nechal tak